tests/test-cases/Group3-Docker-Compose/3-01-Docker-Compose-LEMP.robot

```diff
@@ -24,13 +24,11 @@ Compose LEMP Server
     Should Be Equal As Integers  ${rc}  0
 
     Set Environment Variable  COMPOSE_HTTP_TIMEOUT  300
-    # must set CURL_CA_BUNDLE to work around Compose bug https://github.com/docker/compose/issues/3365
-    Set Environment Variable  CURL_CA_BUNDLE  ${EMPTY}
 
     ${vch_ip}=  Get Environment Variable  VCH_IP  %{VCH-IP}
     Log To Console  \nThe VCH IP is %{VCH-IP}
 
     Run  cat %{GOPATH}/src/github.com/vmware/vic/demos/compose/webserving-app/docker-compose.yml | sed -e "s/192.168.60.130/${vch_ip}/g" > lemp-compose.yml
-    ${rc}  ${output}=  Run And Return Rc And Output  docker-compose %{VCH-PARAMS} --file lemp-compose.yml up -d
+    ${rc}  ${output}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} --file lemp-compose.yml up -d
```

Comment:
As best I can determine `COMPOSE-PARAMS` are synonymous with `VCH-PARAMS` other than the presence of `--tls` in the case where we have client certificates, and it can be dropped in that case for docker client. Please just make `VCH-PARAMS` suited to compose so we don't have this duplication.